### DISCUSSION TITLE
Alerting: With viewers_can_edit enabled the users are not able to create silences

### DISCUSSION THREAD

**Discussion by paulojmdias** [grafana/grafana#73470] [Alerting]
### What happened?

Even with `viewers_can_edit` with the `True` value, the users can't create silences using Grafana Alerting. We can only achieve that if we give the `Editor` role to the users.

Discussion started here: https://community.grafana.com/t/allow-viewers-to-silence-their-own-alert/73955

### What did you expect to happen?

Users with the `Viewer` role and configuration `viewers_can_edit` with the `True` value can create silences. 

### Did this work before?

No.

### How do we reproduce it?

1. Make the `viewers_can_edit` with the `True` value in the configuration.
2. Create a user with the `Viewer` role.
3. The silence button will not appear in the `Alerting` -> `Silences` panel

### Environment (with versions)?

Grafana: 9.5.7
OS: Ubuntu 22
Browser: Chrome

### Grafana platform?

Docker

**Comment by armandgrillet** [comment]
`viewers_can_edit` can make temporary edits but cannot save them. This does not strictly relate to being able to create silences. This is a feature request, more fine grained access control is available in Grafana Enterprise.

**Comment by armandgrillet** [comment]
Moved back to [an issue](https://github.com/grafana/grafana/issues/73911) as we now use issues for feature requests, sorry for the back and forth.
